Sources/ChatUI/Essentials/GiphyConfiguration.swift

```diff
@@ -7,18 +7,21 @@ public struct GiphyConfiguration {
     public let shouldLocalizeSearch: Bool
     public let mediaTypeConfig: [GiphyUISDK.GPHContentType]
     public let presentationDetents: CGFloat
+    public let showAttributionMark: Bool
```

Comment:
```suggestion
    public let showsAttributionMark: Bool
```
I recommend that you name the boolean type property as 3rd person singular verb.
Also line#6
```swift
- showConfirmationScreen: Bool
+ showsConfirmationScreen: Bool
```

Reply:
Hi I pushed this commit accidentally, it should be a separate pull request, https://support.giphy.com/hc/en-us/articles/360035158592-What-conditions-does-my-app-project-need-to-meet-in-order-to-get-a-production-API-Key-

Reply:
is there any discourse chat server or other chat way to discuss any release dates for the OSS project? thanks for creating this project